package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "nyc lerna run test",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "cleanInstall": "lerna exec npm install --ignore-scripts",
+    "cleanInstall": "lerna exec yarn install --ignore-scripts",
```

Comment:
❤️ . 

@darahayes Any voice against it?

Reply:
Coming to this late but go for it :)